blogposts/announcing-sourcegraph-3.15.md

```diff
@@ -0,0 +1,309 @@
+
+Enable this feature by adding `"permissions.backgroundSync": {"enabled": true}` in your site configuration settings.
+
+## Experimental AND/OR operators for search contents
+
+TODO: Video
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Search for file contents using and- and or-expressions in queries. These operators enable more expressive searches in regexp and structural search modes. [The documentation](https://docs.sourcegraph.com/user/search/queries#operators) includes more details about how to these operators and their precedence and grouping. 
+
+For example, if you want to find all of the places where two functions are within the same file, you can use the AND operator:
+
+EXAMPLE
+
+Or, if you want to find everywhere two functions are used, whether or not the other is present, you would use the OR operator:
+
+EXAMPLE
+
+You can even use them together:
+
+EXAMPLE
+
+Currently, operators only support searching file contents and do not apply to search scopes, you will see an alert if your query is not supported. 
+
+Enable this feature by adding `{"experimentalFeatures": {"andOrQuery": "enabled"}}` to your global, organization, or user settings.
+
+## Onboarding prompts contain more context
+
+TODO: video
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+When using Sourcegraph for the first time on a private instance, a getting started section is shown to help admins and users make sure their Sourcegraph instance is properly configured, and that they know how to perform a search. The getting started steps now include more context about how to complete each step.
+
+## Uncrowding the toolbar
+
+TODO: Screenshot comparison
+
+Reworked action bars
+
+## User event logs available in user settings
+
+![user event log](/blog/3-15-user-event-log.png)
+
+Users and site administrators can now view a log of their actions/events in the user settings. This enables site admins to perform rough audits of users and helps users better understand the information Sourcegraph logs.
```

Comment:
We should clarify that we only log information locally on their instance, it's not sent anywhere!

In fact, this is mostly a nice debug tool — I'm not sure if it really provides enough end-user benefit to qualify for the blog post. Not opposed to it, just not sure how the decision is usually made.

```suggestion
Users and site administrators can now view a log of their actions and the events logged in the user settings. This enables site admins to perform audits of user activity, and helps provide users with transparency around the activity that Sourcegraph logs. Note that these logs are only stored locally.
```

Reply:
Good question! In this case I like that we're letting admins know that we're thinking about auditing. Otherwise I agree and would leave out.